content/en/docs/components/training/user-guides/tensorflow.md

```diff
@@ -64,7 +64,8 @@ spec:
                 - --training_steps=1000
 ```
 
-If you want to give your `TFJob` pods access to credentials secrets, such as the GCP credentials automatically created when you do a GKE-based Kubeflow installation, you can mount and use a secret like this:
+If you want to give your `TFJob` pods access to credentials secrets, such as the GCP credentials
```

Comment:
```suggestion
If you want to give your `TFJob` pods access to credentials secrets, such as the Google Cloud credentials
```

Maybe, we should use the Google Cloud instead of GCP based on https://cloud.google.com/blog/en/topics/developers-practitioners/introducing-new-homepage-google-cloud?hl=en.

> Google Cloud Platform is now called Google Cloud
To further simplify your experience and ensure consistency across our products, Google Cloud Platform is now called Google Cloud. In addition, the mobile app name (previously called Cloud Console app) is renamed to Google Cloud app, and you'll see this update reflected across most of our website and documentation.